tests/integration/helper.py

```diff
@@ -19,6 +20,11 @@
 
 import pandas as pd
 
+from kolena._utils import krequests
```

Comment:
could we use `from kolena._utils import krequests_v2 as krequests` at here to avoid specifying `api_version="v2",` in the below?

Reply:
will do